collect_app/src/main/java/org/odk/collect/android/preferences/AdminKeys.java

```diff
@@ -59,8 +59,7 @@
     private static final String KEY_NAVIGATION                  = "change_navigation";
     static final String KEY_CONSTRAINT_BEHAVIOR                 = "change_constraint_behavior";
 
-    private static final String KEY_SHOW_MAP_SDK                = "show_map_sdk";
-    private static final String KEY_SHOW_MAP_BASEMAP            = "show_map_basemap";
+    private static final String KEY_BASEMAP_SOURCE              = "basemap_source";
```

Comment:
It looks like admin keys aren't migrated, yet, right? I think that if either `show_map_sdk` or `show_map_basemap` is set to false in v1.22, the Maps section in v1.23 should be hidden. 

Since `basemap_source` actually hides the full Maps setting category, I wonder whether the key should have a different name.

And finally, the admin settings just show the cryptic "Source" string. It would be helpful if that said "Maps" instead.

I can't exactly remember what's tied to what but am hoping these things are straightforward.

Reply:
Okay, this is all cleaned up now.  I've added the migrations for this.

Reply:
I've manually confirmed that when I turn off either or both of "Show Map SDK Options" or "Show Basemap Options" using an APK at `master`, and then upgrade to this branch, it causes the Maps item to disappear from the General Settings screen and the Maps checkbox to be unchecked in the Admin Settings > User Settings screen.  If I leave these both on, or if I install this branch on a device with no previous APK, the Maps item is visible by default and the Maps checkbox is checked by default on the Admin screen.